1.1/spec/09-Part-06.adoc

```diff
@@ -113,36 +118,48 @@ The class http://www.opengis.net/ont/geosparql#FeatureCollection[`geo:FeatureCol
 
 ```turtle
 geo:FeatureCollection
-  rdf:type owl:Class ;
+  a owl:Class ;
+  rdfs:isDefinedBy geo: ;
+  skos:prefLabel "Collection of geospatial features" ;
+  skos:definition "The class Feature Collection represents 
+                  any collection of individual Features."@en ;  
   rdfs:subClassOf geo:SpatialObjectCollection ;
   rdfs:subClassOf [
-      rdf:type owl:Restriction ;
+      a owl:Restriction ;
       owl:allValuesFrom :Feature ;
       owl:onProperty rdfs:member ;
     ] ;
-  skos:prefLabel "Collection of geospatial features" ;
-  skos:definition "The class Feature Collection represents any collection of individual Features. " ;
 .
 ```
 
 The restriction imposed on the more general <<Class: SpatialObjectCollection, Spatial Object Collection>> that defines this class is that only instances of <<Class: Feature, Feature>> are allowed to be members of it and these are to be indicated with the `rdfs:member` property.
 
 |===
-| *Req 6* Implementations shall allow the RDFS class http://www.opengis.net/ont/geosparql#FeatureCollection[`geo:FeatureCollection`] to be used in SPARQL graph patterns.
+| *Req 6* Implementations shall allow the RDFS class <<Class: geo:FeatureCollection, `geo:FeatureCollection`>> to be used in SPARQL graph patterns.
 |http://www.opengis.net/spec/geosparql/1.1/req/core/feature-collection-class[`http://www.opengis.net/spec/geosparql/1.1/req/core/feature-collection-class`]
 |===
 
 === Standard Properties for geo:SpatialObject
 
-_To be defined_
+Spatial Object is essentially an abstract class and one that is not expected to be instantiated directly but rather used through more concrete subclasses of it. For this reason, no standard properties of Spatial Object are given here, directly. 
+
+Since GeoSPARQL defines two subclasses of Spatial Object, <<Class: geo:Feature, Feature>> & <<Class: geo:Geometry, Geometry>>, the union of their properties may be indirectly considered to be the standard properties of Spatial Object.
 
 === Standard Properties for geo:Feature
 
 Properties are defined for associating geometries with features.
```

Comment:
Not really part of the topic of this PR, but this line no longer covers all properties.